plugins/cad/src/components/Controls/Tabs.tsx

```diff
@@ -75,7 +75,7 @@ export const Tabs = (props: TabsProps) => {
         ))}
       </MUITabs>
       {props.tabs.map(({ content }, index) => (
-        <div className={classes.content} hidden={value !== index}>
+        <div className={classes.content} key={index} hidden={value !== index}>
```

Comment:
Yes, it could have been a concern, but in this case it's acceptable. I based my implementation (including `keys`) on a previously used Backstage component which was removed from the framework.